### PR TITLE
Added

### PR DESCRIPTION
**Pull request by munishaparikh**
just made some change

### CODE REVIEW

File: src/main/resources/public/index.html [hunk 1]
```diff
@@ -436,8 +436,8 @@ <H1 style="float: right;">Theirs</H1>
         <p><a href-"javascript:void(0)" onclick="toggle_visibility('Error_pop_up_box');">Click for Error Popup Box</a></p>
         <p><a href-"javascript:void(0)" onclick="toggle_visibility('ShipsDestroyed_pop_up_box');">Click for Player Ships Destroyed Popup Box</a></p>
     </div>
-<input type="button" value="easy mode">
-<input type="button" value="hard mode">
+<input type="button" value="easy mode!">
+<input type="button" value="hard mode!">
```
Comment (alfayeza) [inline src/main/resources/public/index.html:440]: That's nice. Although if you made the ! only on hard that would be better! Good job though

**Comment by alfayeza** [comment]
Good job, check in line comments.

**Comment by alfayeza** [comment]
Good work on the HTML file, that looks great. Awesome job !

**Comment by munishaparikh** [comment]
Yes, why not. That's what teammates are for!

**Pull request merged**